drivers/hmis/app/models/hmis/hud/custom_assessment.rb

```diff
@@ -4,6 +4,9 @@
 # License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
 ###
 
+# "CustomAssessment" is NOT a HUD record type. Although it uses CamelCase conventions, this model is particular to Open Path
```

Comment:
One small note, that might be worth adding.  These are camel case because they are intended to be compatible with Appendix C **Custom file transfer template** in the HUD HMIS CSV spec.  This specifies optional additional CSV files with the naming convention of `Custom*.csv`